Content.Server/Radio/Components/RadioMicrophoneComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Server.Radio.EntitySystems;
+using Content.Shared.Radio;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Radio.Components;
+
+/// <summary>
+///     Listens for local chat messages and relays them to some radio frequency
+/// </summary>
+[RegisterComponent]
+[Access(typeof(RadioDeviceSystem))]
+public sealed class RadioMicrophoneComponent : Component
+{
+    [DataField("broadcastChannel", customTypeSerializer: typeof(PrototypeIdSerializer<RadioChannelPrototype>))]
+    public string BroadcastChannel = "Common";
+
+    [DataField("listenRange")]
+    public int ListenRange  = 4;
+
+    [DataField("enabled")]
+    public bool Enabled = false;
```

Comment:
VV readwrite

Reply:
range is readwrite now. enabled isn't because that requires more logic to add the active components and I cbf doing that atm.